src/Utility/Assembly.cpp

```diff
@@ -78,9 +78,10 @@ void Assembly::writeInstruction(const llvm::MCInst inst, memory_ostream *stream)
     LogCallback(LogPriority::DEBUG, "Assembly::writeInstruction", [&] (FILE *log) -> void {
         std::string disass;
         llvm::raw_string_ostream disassOs(disass);
-        printDisasm(inst, disassOs);
+        uint64_t address = (uint64_t) stream->get_ptr() + (uint64_t) pos;
```

Comment:
``uintptr_t`` instead of of ``uint64_t``

Reply:
see previous comment.
I change the C cast to an reinterpret_cast.